### DISCUSSION TITLE
Farming  Efficiency (Sorc Level 30-35 Cows)

### DISCUSSION THREAD

**Discussion by ebner20** [isid0re/SoloLeveling#86] [Q&A]
I realize that bot farms Cows before it will move on to Nightmare if below 35.  One thing that I noticed was that when it runs of out mana potions doing cows (which is quite often), it will port to Rouge Encampment, do town heals, then wp to Pandemonium Fortress to buy mana potions, then wp back to Rouge Encampment then take portal back to farming cows.  

Is it possible to get it to do all chores when framing cows in Rouge Encampment?

**Comment by isid0re** [comment]
It's definstely a good idea to save time. at higher difficulties like nightmare and hell. I can see the appeal, since pots dont vary as much.

Implementation wise, it would require quite a bit of changes to kolbots code base.
